cfgov/jinja2/v1/_includes/molecules/link-blob.html

```diff
@@ -0,0 +1,14 @@
+<div class="o-link-blob">
```

Comment:
I'm just removing this file since it's just a duplicate of `half-width-link-blob` and we don't have a spec for a link blob that's non-half-width.